### PR TITLE
Add more testnets

### PR DESCRIPTION
**Pull request by orkunkl**
I left some places empty. We can fill in the details later. Here are the questions to finish this PR

- [x] `Coral` network wasmd and cosmjs versions
- [x] `Gaia-flex` network wasmd and cosmjs versions

### CODE REVIEW

**Comment by ethanfrey** [comment]
Both are wasmd v0.10.0 and cosmjs v0.22.0

We will actually build custom binaries for them with the name of the testnet, but this will just be wasmd with some configuration and in the wasmd repo.

Looking at this, I also thought that the wasmd and cosmjs versions should be text and links from the main page not in the config. Config can just hold the urls and chain id

**Comment by ethanfrey** [comment]
I will try to prepare pre-release versions of corald and gaiaflexd monday to link to.

**Review by ethanfrey — APPROVED** [review]
Proper versions.

Can you pull these out and put them in the README files (as well).

File: coral/config [hunk 1]
```diff
@@ -1,8 +1,8 @@
 CHAIN_ID="coral"
 TESTNET_NAME="coralnet"
 // TODO versions
```
Comment (ethanfrey) [inline coral/config]: You can remove the TODO as well

File: coral/config [hunk 2]
```diff
@@ -0,0 +1,11 @@
+CHAIN_ID="coral"
+TESTNET_NAME="coralnet"
+// TODO versions
```
Comment (orkunkl) [inline coral/config]:
```suggestion
```

File: coral/config [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+WASMD_VERSION="v0.10.0"
+COSMJS_VERSION="v0.22.0"
+
+RPC="https://rpc.coralnet.cosmwasm.com"
+LCD="https://lcd.coralnet.cosmwasm.com"
+FAUCET="https://faucet.coralnet.cosmwasm.com"
+// TODO seed node
```
Comment (orkunkl) [inline coral/config]:
```suggestion
```

File: coral/genesis.json [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+        "denom": "ustake",
+        "amount": "1000"
+      }
+    },
+    "bank": {
+      "send_enabled": true
+    },
+    "genutil": {
+      "gentxs": [
+        // TODO add gentxs
```
Comment (orkunkl) [inline coral/genesis.json]:
```suggestion
```

File: coral/genesis.json [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+    "auth": {
+      "params": {
+        "max_memo_characters": "256",
+        "tx_sig_limit": "7",
+        "tx_size_cost_per_byte": "10",
+        "sig_verify_cost_ed25519": "590",
+        "sig_verify_cost_secp256k1": "1000"
+      },
+      "accounts": [
+        // TODO add accounts with ushell tokens
```
Comment (orkunkl) [inline coral/genesis.json]:
```suggestion
```

File: gaia-flex/config [hunk 6]
```diff
@@ -0,0 +1,11 @@
+CHAIN_ID="gaia-flex"
+TESTNET_NAME="gaia-flexnet"
+// TODO versions
```
Comment (orkunkl) [inline gaia-flex/config]:
```suggestion
```

File: gaia-flex/config [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+WASMD_VERSION="v0.10.0"
+COSMJS_VERSION="v0.22.0"
+
+RPC="https://rpc.gaia-flex.cosmwasm.com"
+LCD="https://lcd.gaia-flex.cosmwasm.com"
+FAUCET="https://faucet.gaia-flex.cosmwasm.com"
+// TODO seed node
```
Comment (orkunkl) [inline gaia-flex/config]:
```suggestion
```

File: gaia-flex/genesis.json [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+        "denom": "ustake",
+        "amount": "1000"
+      }
+    },
+    "bank": {
+      "send_enabled": true
+    },
+    "genutil": {
+      "gentxs": [
+        // TODO add gentxs
```
Comment (orkunkl) [inline gaia-flex/genesis.json]:
```suggestion
```

File: gaia-flex/genesis.json [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+    "auth": {
+      "params": {
+        "max_memo_characters": "256",
+        "tx_sig_limit": "7",
+        "tx_size_cost_per_byte": "10",
+        "sig_verify_cost_ed25519": "590",
+        "sig_verify_cost_secp256k1": "1000"
+      },
+      "accounts": [
+        // TODO add accounts with ushell tokens
```
Comment (orkunkl) [inline gaia-flex/genesis.json]:
```suggestion
```

**Comment by orkunkl** [comment]
Waiting for https://github.com/CosmWasm/wasmd/pull/223

**Comment by orkunkl** [comment]
I assume we need `gaiaflexd` and `gaiaflexcli` as well. If the procedure will be the same as https://github.com/CosmWasm/wasmd/pull/223, I can refer to `make build-gaiaflex` in the document without waiting for the PR? @ethanfrey

**Review by ethanfrey — APPROVED** [review]
Nice

**Comment by ethanfrey** [comment]
Yes, add ref to gaiaflex now. And link to v0.10.0-beta1 if you wish. I hope to add that tonight

File: gaia-flex/README.md [hunk 10]
```diff
@@ -2,9 +2,11 @@
 
 A permissionless network running on `wasd` 0.10
 
-**Started**: TBA
-
-**Ended**: not yet finished
+- **wasmd version**: `v0.10.0-beta1`
```
Comment (ethanfrey) [inline gaia-flex/README.md]: This will be `v0.10.0` as above.

**Review by ethanfrey — APPROVED** [review]
Once change and we can merge

File: gaia-flex/README.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
+# Gaia Flex (WIP)
+
+A permissionless network running on `wasd` 0.10
+
+- **wasmd version**: `v0.10.0-beta1`
```
Comment (orkunkl) [inline gaia-flex/README.md]:
```suggestion
- **wasmd version**: `v0.10.0`
```

**Pull request merged**